bluebinder.c

```diff
@@ -406,7 +406,7 @@ bluebinder_callbacks_transact(
 
             vec = gbinder_reader_read_hidl_vec(&reader, &count, &elemsize);
             if (elemsize != 1) {
-                fprintf(stderr, "Received unexpected array element size, expected sizeof(uint8_t)\n");
+                fprintf(stderr, "Received unexpected array element size, expected sizeof(size_t)\n");
```

Comment:
sizeof(size_t) seems wrong to me. The check above is still for 1, so no need to change error message.

Reply:
okk thanks for review, i'll edit and rebase